### DISCUSSION TITLE
Predictions out of Test and Score

### DISCUSSION THREAD

**Discussion by shiuhng** [biolab/orange3#5178] [Q&A] [answered]
Hello

1)	Refering to widget Test and Score. Does “Test and Score” use the all of the dateset in building the model again and output it as “predictions” after it has gone through with all the folds? 
or was it that the prediction output from Test and score widget a collection of predictions in each of the fold?

**Comment by janezd** [comment, marked as answer]
The latter. If you run cross validation, you get predictions from cross validation.

The former -- building the model from some data and then use it to make predictions on this very same data -- would be very wrong. (The user can of course explicitly require using the same data for training and testing.)

**Reply by shiuhng** [reply]
With Thx
